src/main/java/org/myrobotlab/service/JMonkeyEngine.java

```diff
@@ -2189,6 +2205,36 @@ public String toJson(Node node) {
     // save it out
   }
 
+  public void setTransform(String name, double x, double y, double z, double xRot, double yRot, double zRot) {
+    addMsg("setTransform", name, x, y, z, xRot, yRot, zRot);
+  }
+
+  public void setTranslation(String name, double x, double y, double z) {
+    addMsg("setTranslation", name, x, y, z);
+  }
+
+  public void setRotation(String name, double xRot, double yRot, double zRot) {
+    addMsg("setRotation", name, xRot, yRot, zRot);
+  }
+
+  public Map<String, String[]> getMultiMapped() {
+    return multiMapped;
+  }
+
+  @Override
+  public void onJointAngles(Map<String, Double> angleMap) {
+    for (String name : angleMap.keySet()) {
+      ServiceInterface si = Runtime.getService(name);
+      if (si instanceof Servo) {
+        ((Servo) si).moveTo(angleMap.get(name));
+      }
+    }
+  }
+
+  public void setDefaultServoSpeed(Double speed) {

```

Comment:
I'm surprised this method is here.. do we need this? is it some sort of helper function that might be better suited on some other service?

Reply:
JMonkey has a concept of a virtual servo driven by servo controls ... so it needs a concept of what the "virtual" speed is - not sure .. don't remember the details .. not in that area yet ... :)